src/Phinx/Console/Command/SeedRun.php

```diff
@@ -104,7 +105,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         // run the seed(ers)
         $start = microtime(true);
-        $this->getManager()->seed($environment, $seed);
+        foreach($seedSet as $seed) {
+            $this->getManager()->seed($environment, trim($seed));
```

Comment:
Are you using the `-s` option to try to run all seeds or something odd? I don't see an error when I just run `phinx seed:run` on the latest `dev-master`.